### ISSUE TITLE
api_url and console_url no longer populated ?

### ISSUE THREAD

**Issue by oybed** [terraform-redhat/terraform-provider-rhcs#344]
In the provider f.k.a. ocm, we were able to access `api_url` and `console_url`, but these are no longer being populated as of rhcs v1.3.0. Is this intentional, or a bug?

OCM v1.0.4 output:
```
resource "ocm_cluster_rosa_classic" "openshift" {
    api_url              = "https://api.<redacted>:6443"
  :
    console_url          = "https://console-openshift-console.apps.<redacted>"
  : 
```

**Comment by bardielle** [comment]
Hello @oybed, can you share your resource configuration? 
I couldn't reproduce it- when I'm creating a cluster I get the `api_url` and `console_url` : 
```resource "rhcs_cluster_rosa_classic" "rosa_sts_cluster" {
  :
    api_url                  = "https://api.terr-cluster.t0zc.p1.openshiftapps.com:6443"
  :
    console_url              = "https://console-openshift-console.apps.terr-cluster.t0zc.p1.openshiftapps.com"
  :
```

**Comment by bardielle** [comment]
I reproduced it using the v1.3.0 of the provider. 
The main issue is that the cluster's attribute is populated before the `rhcs_cluster_rosa_classic` resource becomes ready. 
Even if you're using the [waiter resource](https://registry.terraform.io/providers/terraform-redhat/rhcs/latest/docs/resources/cluster_wait), the cluster resource state (which is a different one) doesn't refresh at all. 

In order to solve it you can choose one of those options:
1. run `terraform refresh` just after the cluster becomes in `ready` state 
2. create `oidc-configuration` before creating the cluster and user the GA version 1.4.0** which includes a waiter attribute as part of the `rhcs_cluster_rosa_classic` itself (https://registry.terraform.io/providers/terraform-redhat/rhcs/latest/docs/resources/cluster_rosa_classic#wait_for_create_complete) - but this is valid only in oidc-configuration   



** the v 1.4.0 will be officially released tomorrow, till then you can use the prerelease version like that: 
```
terraform {
  required_providers {

    rhcs = {
      version = "1.4.0-prerelease.4"
      source  = "terraform-redhat/rhcs"
    }
  }
}

```

**Comment by bardielle** [comment]
@oybed, we already released v1.4.0. 
Can you reproduce it and update/ close the issue?

**Comment by andyrepton** [comment]
Hi @bardielle , I can confirm that by moving to the `wait_for_create_complete` option this fixes this bug. Tested with version v1.5.0
